### ISSUE TITLE
模型训练的时候有问题

### ISSUE THREAD

**Issue by cdj0311** [xccds/chinese_wordseg_keras#2]
在模型结构中出现问题：
model = Sequential()
model.add(Embedding(maxfeatures, 100,weights=init_weight)) 
model.add(Flatten())
model.add(Dense(700, 100))
model.add(Dropout(0.5))
model.add(Activation('relu'))
model.add(Dense(100, nb_classes))
model.add(Activation('softmax'))
model.compile(loss='categorical_crossentropy', optimizer='adam')

其中model.add(Flatten())有错：
Exception: The shape of the input to "Flatten" is not fully defined (got (None, 100). Make sure to pass a complete "input_shape" or "batch_input_shape" argument to the first layer in your model.

**Comment by emmapapa** [comment]
你这个问题有没有解决呢？我也出现了这个问题 TT

**Comment by bobeina** [comment]
看人家说是karas新版本调用方法变了，就算这里的解决了，后面的代码有些也要跟着改
——正在改的人留
